js/grid.custom.js

```diff
@@ -417,7 +417,7 @@ $.jgrid.extend({
 								break;
 							}
 						}
-						var st = soptions.searchtitle !== null ? soptions.searchtitle : p.operandTitle;
+						var st = soptions.searchtitle != null ? soptions.searchtitle : p.operandTitle;
```

Comment:
I think we can use !== operator if we are certain that undefined is always.
I'm not sure, i don't know all the code, so i used != null (that test null, undefined, empty string...)

I think !== is better than != in most cases, but sometimes != is better to avoid && or || in if statements.
If it's just a warning, i think it's not blocking for build, is it ?